### ISSUE TITLE
Is there a way to use the front camera as the scanning camera?

### ISSUE THREAD

**Issue by theekshanak** [wu9007/qrcode_scanner#77]

**Comment by yoshuaaadk** [comment]
i also have same question, is this "qrscan" package can use front camera? thanks. I hope somebody answer this :')

**Comment by poka-IT** [comment]
Same here, this should be wonderful to have the possibility to use front camera.

**Comment by wu9007** [comment]
@theekshanak no. `SecondActivity.startCamera()` binds `CameraSelector.DEFAULT_BACK_CAMERA` only. iOS `ScanViewController` uses `.builtInWideAngleCamera` / `.back`.

qrscan is a one-shot full-screen Activity, not a widget with a camera-facing argument. Front camera will not be added. For a selfie / in-widget scanner use `mobile_scanner`.

Closing as not planned. Same ask as #124.
